aptos/contracts/sources/Governance.move

```diff
@@ -75,6 +71,39 @@ module Wormhole::Governance {
         }
     }
 
+    #[test]
+    public fun test_parse_guardian_set_upgrade() {
+        use Wormhole::u32;
+        use Wormhole::Structs::{createGuardian};
+
+        let b = x"00000000000000000000000000000000000000000000000000000000436f7265020000000000011358cc3ae5c097b213ce3c81979e1b9f9570746aa5ff6cb952589bde862c25ef4392132fb9d4a42157114de8460193bdf3a2fcf81f86a09765f4762fd1107a0086b32d7a0977926a205131d8731d39cbeb8c82b2fd82faed2711d59af0f2499d16e726f6b211b39756c042441be6d8650b69b54ebe715e234354ce5b4d348fb74b958e8966e2ec3dbd4958a7cdeb5f7389fa26941519f0863349c223b73a6ddee774a3bf913953d695260d88bc1aa25a4eee363ef0000ac0076727b35fbea2dac28fee5ccb0fea768eaf45ced136b9d9e24903464ae889f5c8a723fc14f93124b7c738843cbb89e864c862c38cddcccf95d2cc37a4dc036a8d232b48f62cdd4731412f4890da798f6896a3331f64b48c12d1d57fd9cbe7081171aa1be1d36cafe3867910f99c09e347899c19c38192b6e7387ccd768277c17dab1b7a5027c0b3cf178e21ad2e77ae06711549cfbb1f9c7a9d8096e85e1487f35515d02a92753504a8d75471b9f49edb6fbebc898f403e4773e95feb15e80c9a99c8348d";
```

Comment:
This test is adapted from
https://github.com/certusone/wormhole/tree/27f479feccb46c4846b36124ab4f67282806ff6a/clients/js/parse_tests/guardian-set-upgrade-1.test and the corresponding https://github.com/certusone/wormhole/tree/27f479feccb46c4846b36124ab4f67282806ff6a/clients/js/parse_tests/guardian-set-upgrade-1.expected

moving forward we should have such a test for each parser.